orchestration/src/test/resources/filteringLooseRequest.json

```diff
@@ -17,10 +17,7 @@
         "template": [
           {
             "role": "user",
-            "content": [ {
-              "type" : "text",
-              "text" : "Hello World! Why is this phrase so famous?"
-            } ]
+            "content": "Hello World! Why is this phrase so famous?"
```

Comment:
(Minor/Question)

Extending my question above, no need for drive-by. I would happily review this as separate PR :)